src/store/actions/api.actions.js

```diff
@@ -313,8 +313,8 @@ export const saveEndpoint = api => dispatch =>
 export const updateEndpoint = api => dispatch =>
     dispatch(openConfirmationModal('update', api, api.name));
 
-export const deleteEndpoint = apiName => dispatch =>
-    dispatch(openConfirmationModal('delete', {}, apiName));
+export const deleteEndpoint = (api, shouldRedirect/*: Boolean*/) => dispatch =>
```

Comment:
why not `isRedirect`? also why not let the redirection be handle as a separate action?

Reply:
+1 for `isRedirect`

Reply:
👨‍⚖️  : "The decision accepted!" 🔨